src/constants.ts

```diff
@@ -103,7 +109,8 @@ export const CONFIRMATION_PERSON_REMOVED_URL = "/confirmation-person-removed";
 export const CONFIRMATION_PERSON_ADDED = "/confirmation-person-added";
 export const MANAGE_AUTHORISED_PEOPLE_CONFIRMATION_PERSON_REMOVED_URL = `${MANAGE_AUTHORISED_PEOPLE_URL}${CONFIRMATION_PERSON_REMOVED_URL}`;
 export const YOUR_COMPANIES_MANAGE_AUTHORISED_PEOPLE_CONFIRMATION_PERSON_REMOVED_URL = `${LANDING_URL}${MANAGE_AUTHORISED_PEOPLE_CONFIRMATION_PERSON_REMOVED_URL}`;
-
+export const REMOVE_COMPANY_URL = `/${REMOVE_COMPANY_PAGE}/:${COMPANY_NUMBER}`;
+export const YOUR_COMPANIES_REMOVE_COMPANY_URL = `${LANDING_URL}/${REMOVE_COMPANY_PAGE}/:${COMPANY_NUMBER}`;
```

Comment:
Why not `${LANDING_URL}${REMOVE_COMPANY_URL}` ?

Reply:
🤦 this went over our heads, nice spot